docs/src/modules/concepts/pages/multi-region.adoc

```diff
@@ -48,16 +68,16 @@ image:geo-e.svg[Geo data replication, width=600]
 
 Meanwhile, all requests made by Alice on her data are handled locally image:steps-1.svg[width=20] and get an immediate reply image:steps-2.svg[width=20]. The write operations are being replicated to the UK image:steps-3.svg[width=20].
 
-=== Primary selection
+== Replicated writes
 
-How Akka assigns the primary region to an entity is configurable. The two main modes are **static**, and **dynamic**.
+The replicated write replication strategy allows every region to be capable of handling writes for all entities. This is done through the use of CRDTs, which can be modified concurrently in different regions, and their changes safely merged without conflict.
 
-In the **static primary selection** mode (which is the default), the primary region for an entity is selected statically as part of the deployment, so all entities have the same primary region. This is useful for scenarios where you want one primary region, with the ability to fail over to another region in the case of a regional outage.
+== Replication Guarantees
```

Comment:
this is a new section that explains how it works, mostly to build trust that the mechanism is sane and solid